addons/product/models/product_pricelist.py

```diff
@@ -583,6 +584,7 @@ def _compute_price(self, price, price_uom, product, quantity=1.0, partner=False)
            The unused parameters are there to make the full context available for overrides.
         """
         self.ensure_one()
+        date = self._context.get('date') or fields.Date.today()
```

Comment:
```suggestion
        date = self.env.context.get('date') or fields.Date.today()
```
Cosmetic change, but I prefer when we use the public variable instead of the private one ;)